test/test_jit_fuser_te.py

```diff
@@ -737,6 +737,22 @@ def foo(hx, cx):
                 # XXX: TE fuser can handle concats in a fusion group.
                 # FileCheck().check("FusedConcat").check_next("return").run(str(graph))
 
+    def test_stack(self):
+        # "aten::stack fusion is not enabled yet with dynamic shapes"
```

Comment:
What is the reason for this restriction?

Reply:
The shape computation formula for aten::stack in JIT is not added in this PR. Will do it in the next PR. It seems a bit compound to have both changes for static and dynamic shapes in one PR.